app/models/view/reports/Recipient.java

```diff
@@ -29,14 +27,7 @@
  *
  * @author Christian Briones (cbriones at dropbox dot com)
  */
-@JsonTypeInfo(
-        use = JsonTypeInfo.Id.NAME,
-        include = JsonTypeInfo.As.PROPERTY,
-        property = "type")
-@JsonSubTypes({
-        @JsonSubTypes.Type(value = EmailRecipient.class, name = "EMAIL"),
-})
-public abstract class Recipient {
+public class Recipient {
```

Comment:
Are there any changes necessary the UI with this change in polymorphism of the view model recipient?